src/app/about/page.js

```diff
@@ -2,15 +2,26 @@
 import Logo from "../../../public/assets/logo.svg";
 import Image from "next/image";
 import { sections } from "@/data/about/Sections.js";
+import Sun from "@/public/assets/sun.svg";
 
 const About = () => {
   return (
-    <div className="pb-56">
+    <div className="pb-56 absolute overflow-x-hidden">
       <Image
         src={Logo}
         alt="ISA Icon"
         className="py-8 mx-auto h-[400px] w-[400px]"
       />
+      <Image
+        src={Sun}
+        alt="blue sun"
+        className="absolute bottom-0 -left-[250px] h-[800px] w-[600px]"
```

Comment:
- you can use tailwind scale to size the suns without using larger values
- make the suns a little bigger